app/presenters/candidate_interface/personal_details_review_presenter.rb

```diff
@@ -82,6 +95,17 @@ def english_language_details_row
       }
     end
 
+    def right_to_work_row
+      if @nationalities_form.first_nationality != 'British' && @nationalities_form.first_nationality != 'Irish'
```

Comment:
What about `UK_AND_IRISH_NATIONALITIES` in `app/forms/candidate_interface/nationalities_form.rb`?